how-we-work/tracking-time.md

```diff
@@ -1,18 +1,77 @@
 # Tracking time
 
-In this section we want to provide a cue on how to correctly calculate working hours, no matter 
-whether they're spent on clients or internal projects.
+A central matter for our everyday work is the ability to make good use of the
+time we spend at the office. To spend time efficiently, measuring time is
+essential.
 
-## The working hours
+During our workday we measure cumulative hours:
 
-The effort on a project is established as number of hours or days spent on a project each week. It 
+* spent in the office;
+* worked on a project for a client.
```

Comment:
we should refer to a "project" in a general way because you may have an internal project assigned for your workday.